src/PKSim.Core/Snapshots/ExpressionContainer.cs

```diff
@@ -0,0 +1,9 @@
+﻿using PKSim.Core.Model;
+
+namespace PKSim.Core.Snapshots
+{
+   public class ExpressionContainer : Parameter
+   {
+      public MembraneLocation? MembraneLocation { get; set; }
```

Comment:
This is the actual Mapping. Membrane location needs to be saved for Transporter container